plugin.video.ign_com/addon.xml

```diff
@@ -13,21 +13,20 @@
         <provides>video</provides>
     </extension>
     <extension point="xbmc.addon.metadata">
-        <summary lang="en">Game and Movie trailers/reviews/news</summary>
-        <description lang="en">Game and Movie trailers/reviews/news</description>
+        <summary lang="en_GB">Game and Movie trailers/reviews/news</summary>
+            <description lang="en_GB">Game and Movie trailers/reviews/news</description>
+            <disclaimer lang="en_GB">For bugs, requests or general questions visit the IGN_COM thread on the Kodi forum.</disclaimer>
+        <language>en_GB</language>
```

Comment:
I don't think this is correct.

Reply:
what's wrong with it?

Reply:
I think `en` is correct, but haven't checked Kodi core code.

Reply:
ah ok, i'll make another pr, Thanks

Reply:
Can't you update the existing PR?